packages/icons-workflow/bin/build-icons-mapping.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+import { getComponentName } from './build.js';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+// builds and saves the icons mapping to a json file
+const saveIconsMapping = async () => {
+    // check if there is file with .csv extension in the bin folder
+    const files = fs.readdirSync(path.join(__dirname, './'));
+    let data = '';
+    files.forEach((file) => {
```

Comment:
nit: we could use `for ... of` instead of `forEach`. this would allow for early termination when we've found the `.csv` file we need, instead of iterating over all files